src/schemas/Program/index.js

```diff
@@ -212,7 +212,7 @@ const typeDefs = gql`
 `;
 
 /* =========
-    Convert GRPC Response to GQL output
+HTTP resolvers
  * ========= */
 const getIsoDate = (time) => (time ? new Date(parseInt(time) * 1000).toISOString() : null);
```

Comment:
will remove the rest of GRPC parts later. Need to keep this to prevent errors when testing the existing endpoints.